website/docs/r/traffic_manager_profile.html.markdown

```diff
@@ -97,6 +100,12 @@ The `monitor_config` block supports:
 
 * `path` - (Optional) The path used by the monitoring checks. Required when `protocol` is set to `HTTP` or `HTTPS` - cannot be set when `protocol` is set to `TCP`.
 
+* `interval_in_seconds` - (Optional) The interval used to check the endpoint health from a Traffic Manager probing agent. You can specify two values here: 30 seconds (normal probing) and 10 seconds (fast probing). If no values are provided, the profile sets to a default value of 30 seconds.
+
+* `timeout_in_seconds` - (Optional) The amount of time the Traffic Manager probing agent should wait before considering that check a failure when a health check probe is sent to the endpoint. If the Probing Interval is set to 30 seconds, then you can set the Timeout value between 5 and 10 seconds. If no value is specified, it uses a default value of 10 seconds. If the Probing Interval is set to 10 seconds, then you can set the Timeout value between 5 and 9 seconds. If no Timeout value is specified, it uses a default value of 9 seconds.
```

Comment:
```suggestion
* `timeout_in_seconds` - (Optional) The amount of time the Traffic Manager probing agent should wait before considering that check a failure when a health check probe is sent to the endpoint. If the Probing Interval is set to `30` seconds, then you can set the Timeout value between `5` and `10` seconds. If no value is specified, it uses a default value of `10` seconds. If the Probing Interval is set to `10` seconds, then you can set the Timeout value between `5` and `9` seconds. If no Timeout value is specified, it uses a default value of `9` seconds.
```

Reply:
Also adding some validation to enforce this would be nice.

Reply:
Is there any example available to validate a property of scheme based on the value of another property? I couldn't find such example, and not sure if it would be feasible here, as for the optional parameters I am relying on the default defined by Azure rather then replicate them here.

Reply:
unfortunately that's not possible at this time (whilst it's possible via `customizeDiff`, that won't work if the fields interpolated) - as such the best place to do that is within the Create/Update function at this point in time